packages/purpose-process/src/services/purposeService.ts

```diff
@@ -1101,7 +1085,15 @@ export function purposeServiceBuilder(
       const eserviceId: EServiceId = unsafeBrandId(seed.eServiceId);
       const consumerId: TenantId = unsafeBrandId(seed.consumerId);
 
-      assertOrganizationIsAConsumer(organizationId, consumerId);
+      assertRequesterCanActAsConsumer(
```

Comment:
Also here for reverse purpose creation you should probably add the delegationId to the created purpose below